tests/test_css_rewriting.py

```diff
@@ -19,15 +19,15 @@
         ),
         ContentForTests(
             b"p { width= } div { background: url(http://exemple.com/img.png)}",
-            b"p { width= } div { background: url(exemple.com/img.png)}",
+            b"p { width= } div { background: url(../exemple.com/img.png)}",
```

Comment:
I'm surprised by this change.

The relative url `exemple.com/img.png` relative to `https://kiwix.org` gives 
```js
> new URL("exemple.com/img.png", "https://kiwix.org").href
"https://kiwix.org/exemple.com/img.png"
```
and this is what we want.
The `../` in front of the new testcase works as an extra `../` is a noop if we are already at root, but it seems wrong anyway.

In any case, one of the version is wrong and it was working pretty well with the previous version. So you will have to give me more argument than `some where assuming invalid values` in the PR description 😉

Reply:
I agree that one of the version is wrong, and I can give more arguments of why I consider the new one is the correct one (and it's not because I wrote it 🤣, and I'm quite sure - even I do not have an example here and I'm lazy to rebuild one - it was not working before)

Here we are rewriting CSS for an article whose URL was "http://kiwix.org" (default value for `ContentForTests`).

The article will hence be stored in the ZIM at `kiwix.org/` path.

The CSS refer to a background URL at `http://exemple.com/img.png`. Inside the ZIM, this image will be stored at `exemple.com/img.png` path.

If serving with kiwix serve, this means that the article will be served at `http(s)://<kiwix_serve_host>/content/<zim_name>/kiwix.org/` and the image at `http(s)://<kiwix_serve_host>/content/<zim_name>/exemple.com/img.png`.

If CSS refer to `../exemple.com/img.png` I think it will be ok (image will be found), not if CSS refer to `exemple.com/img.png`.

Reply:
Edit: I think it was working in fact sometimes in the past in fact, the root cause of this change is linked to the change that both `http://kiwix.org` and `http://kiwix.org/` must be stored at `kiwix.org/`(and are now stored at this path). Test was ok before because `http://kiwix.org` was badly stored at `kiwix.org`

Reply:
Ok. I have my explanation now 🙂  Thanks.

For reference (confirming what you say):
```js
>> new URL("exemple.com/img", "http://kiwix_server_host.com/root_prefix/content/zim_name/kiwix.org").href
"http://kiwix_server_host.com/root_prefix/content/zim_name/exemple.com/img"  // GOOD
>> new URL("exemple.com/img", "http://kiwix_server_host.com/root_prefix/content/zim_name/kiwix.org/").href
"http://kiwix_server_host.com/root_prefix/content/zim_name/kiwix.org/exemple.com/img" // BAD
>> new URL("../exemple.com/img", "http://kiwix_server_host.com/root_prefix/content/zim_name/kiwix.org").href
"http://kiwix_server_host.com/root_prefix/content/exemple.com/img" // BAD
>> new URL("../exemple.com/img", "http://kiwix_server_host.com/root_prefix/content/zim_name/kiwix.org/").href
"http://kiwix_server_host.com/root_prefix/content/zim_name/exemple.com/img"  // GOOD
```